archspec/cpu/cpuid_flags.json

```diff
@@ -0,0 +1,622 @@
+    "eax": 1,
+    "ecx": 15
+  },
+  "pcid": {
+    "eax": 1,
+    "ecx": 17
+  },
+  "dca": {
+    "eax": 1,
+    "ecx": 18
+  },
+  "sse4_1": {
+    "eax": 1,
+    "ecx": 19
+  },
+  "sse4_2": {
+    "eax": 1,
+    "ecx": 20
+  },
+  "x2apic": {
+    "eax": 1,
+    "ecx": 21
+  },
+  "movbe": {
+    "eax": 1,
+    "ecx": 22
+  },
+  "popcnt": {
+    "eax": 1,
+    "ecx": 23
+  },
+  "tscdeadline": {
+    "eax": 1,
+    "ecx": 24
+  },
+  "aes": {
+    "eax": 1,
+    "ecx": 25
+  },
+  "xsave": {
+    "eax": 1,
+    "ecx": 26
+  },
+  "osxsave": {
+    "eax": 1,
+    "ecx": 27
+  },
+  "avx": {
+    "eax": 1,
+    "ecx": 28
+  },
+  "f16c": {
+    "eax": 1,
+    "ecx": 29
+  },
+  "rdrnd": {
+    "eax": 1,
+    "ecx": 30
+  },
+  "hypervisor": {
+    "eax": 1,
+    "ecx": 31
+  },
+  "sgx": {
+    "eax": 7,
+    "ebx": 2
+  },
+  "bmi1": {
+    "eax": 7,
+    "ebx": 3
+  },
+  "hle": {
+    "eax": 7,
+    "ebx": 4
+  },
+  "avx2": {
+    "eax": 7,
+    "ebx": 5
+  },
+  "smep": {
+    "eax": 7,
+    "ebx": 7
+  },
+  "bmi2": {
+    "eax": 7,
+    "ebx": 8
+  },
+  "erms": {
+    "eax": 7,
+    "ebx": 9
+  },
+  "invpcid": {
+    "eax": 7,
+    "ebx": 10
+  },
+  "rtm": {
+    "eax": 7,
+    "ebx": 11
+  },
+  "pqm": {
+    "eax": 7,
+    "ebx": 12
+  },
+  "mpx": {
+    "eax": 7,
+    "ebx": 14
+  },
+  "pqe": {
+    "eax": 7,
+    "ebx": 15
+  },
+  "avx512f": {
+    "eax": 7,
+    "ebx": 16
+  },
+  "avx512dq": {
+    "eax": 7,
+    "ebx": 17
+  },
+  "rdseed": {
+    "eax": 7,
+    "ebx": 18
+  },
+  "adx": {
+    "eax": 7,
+    "ebx": 19
+  },
+  "smap": {
+    "eax": 7,
+    "ebx": 20
+  },
+  "avx512ifma": {
+    "eax": 7,
+    "ebx": 21
+  },
+  "pcommit": {
+    "eax": 7,
+    "ebx": 22
+  },
+  "clflushopt": {
+    "eax": 7,
+    "ebx": 23
+  },
+  "clwb": {
+    "eax": 7,
+    "ebx": 24
+  },
+  "intel_pt": {
+    "eax": 7,
+    "ebx": 25
+  },
+  "avx512pf": {
+    "eax": 7,
+    "ebx": 26
+  },
+  "avx512er": {
+    "eax": 7,
+    "ebx": 27
+  },
+  "avx512cd": {
+    "eax": 7,
+    "ebx": 28
+  },
+  "sha": {
+    "eax": 7,
+    "ebx": 29
+  },
+  "avx512bw": {
+    "eax": 7,
+    "ebx": 30
+  },
+  "avx512vl": {
+    "eax": 7,
+    "ebx": 31
+  },
+  "prefetchwt1": {
+    "eax": 7,
+    "ecx": 0
+  },
+  "avx512vbmi": {
+    "eax": 7,
+    "ecx": 1
+  },
+  "umip": {
+    "eax": 7,
+    "ecx": 2
+  },
+  "pku": {
+    "eax": 7,
+    "ecx": 3
+  },
+  "ospke": {
+    "eax": 7,
+    "ecx": 4
+  },
+  "avx512vbmi2": {
+    "eax": 7,
+    "ecx": 6
+  },
+  "gfni": {
+    "eax": 7,
+    "ecx": 8
+  },
+  "vaes": {
+    "eax": 7,
+    "ecx": 9
+  },
+  "vpclmulqdq": {
+    "eax": 7,
+    "ecx": 10
+  },
+  "avx512vnni": {
+    "eax": 7,
+    "ecx": 11
+  },
+  "avx512bitalg": {
+    "eax": 7,
+    "ecx": 12
+  },
+  "avx512vpopcntdq": {
+    "eax": 7,
+    "ecx": 14
+  },
+  "rdpid": {
+    "eax": 7,
+    "ecx": 22
+  },
+  "sgx_lc": {
+    "eax": 7,
+    "ecx": 30
+  },
+  "fpu": {
+    "eax": 2147483649,
+    "ebx": 0
+  },
```

Comment:
Looking at https://en.wikipedia.org/wiki/CPUID#EAX=80000001h:_Extended_Processor_Info_and_Feature_Bits I think this and the items below are wrong. The bug comes directly from `cpuinfo`, where `ebx` should really be `edx`.